### PR TITLE
create pipeline

### PR DESCRIPTION
**Pull request by joejstuart**

### CODE REVIEW

File: kubevirt-ami-centos.json [hunk 1]
```diff
@@ -38,6 +39,11 @@
       "source": "image-files/virtctl",
       "destination": "/home/centos/"
     },
+    {
+      "type": "file",
+      "source": "tests",
+      "destination": "/home/centos"
+    },
```
Comment (rwsu) [inline kubevirt-ami-centos.json]: The test playbook uploads the test. It is a duplicate step. But eventually I think we will move it here to make the tests/manifests a permanent part of the image.

File: kubevirt-ami-centos.json [hunk 2]
```diff
@@ -16,7 +17,7 @@
     "source_ami": "{{user `source_ami`}}",
     "instance_type": "{{user `instance_type`}}",
     "ssh_username": "centos",
-    "ami_name": "centos-kubevirt-{{timestamp}}",
+    "ami_name": "{{user `aws_instance_name`}}",
```
Comment (rwsu) [inline kubevirt-ami-centos.json]: Is the instance name kubevirt-centos-PR-X or kubevirt-centos-build-X?

**Comment by rwsu** [comment]
@joejstuart I was checking out the AWS console, and noticed that when the test fails, the EC2 test instance doesn't get terminated. I think we will need to move the instance termination out of the test playbook and into a separate command so it doesn't get skipped.

File: kubevirt-ami-centos.json (hunk 2 above)
Comment (joejstuart) [reply kubevirt-ami-centos.json]: I've made it kubevirt-centos-PR-X if a pull request triggers the build. The build-X was what I named it when first testing.

**Comment by rwsu** [comment]
I just merged Karim's PR to fix the manifest crd issue which solves the Kubernetes v1.11 compatiblity problem.

There are still a couple of outstanding issues that will prevent the test from passing.
1. https://github.com/kubevirt/kubevirt/issues/1346
2. https://github.com/kubevirt/kubevirt/issues/1316

Issue 1 has a fix that has been merged, but we will need it to be included in next release for us to pick it up.
Issue 2 is still unresolved.

**Comment by rwsu** [comment]
Thanks for your contribution!

**Pull request merged**